dvc/commands/experiments/gc.py

```diff
@@ -110,6 +120,17 @@ def add_parser(experiments_subparsers, parent_parser):
         default=False,
         help="Keep experiments derived from all Git commits.",
     )
+    experiments_gc_parser.add_argument(
+        "--date",
+        type=str,
+        dest="commit_date",
+        metavar="<commit_date>",
```

Comment:
How about `<yyyymmdd>` or something that gives an idea of the type/format expected? `--data <date>` is not very informative.